docs/publishing.md

```diff
@@ -12,76 +12,68 @@ version bumped.
-4. Create a pull request for these changes. When reviewing the PR check version of compile assets and check release notes for the changes made.
+8. Create a pull request and copy the changelog text. 
+   When reviewing the PR, check that the version numbers have been updated and that the compiled assets use this verson number.
 
-5. Once the pull request is approved, merge to **master** and run `npm run release` in **master**. 
+9. Once the pull request is approved, merge to **master**. 
 
-  This will:
-  - check if you're logged in to NPM as the correct user. If not, it will abort.
-  - run `npm publish` for each package if the package hasn't been published yet
-    (check that the currect version matches the version on NPM)
-  - run `git tag v<version>` to create a new tag if current git tag doesn't match latest published tag 
-  - push the tag to remote with `git push --tags`
-  - create a zip file of the `dist` directory with
-    `git archive -o ./release-<version>.zip HEAD:dist`
+10. Checkout **master** and pull the latest changes.
 
-6. Create a release in Github interface, ddd release notes to and attach the ZIP. 
+11. Log into npm, using team [credentials](https://github.com/alphagov/design-system-team-credentials/tree/master/npm/govuk-patterns-and-tools).
 
-7. Add Trello cards to "This Sprint" column for
-  - updating GOV.UK Design System to use the latest releasex
-  - updating the Prototype Kit to use the latest release 
+12. Run `npm run release`.
 
+  This will:
+  - check that you're logged in to npm as the correct user.
+  - publish each package if the package has not been published yet
+  - create a new tag if the current git tag does not match the latest published tag 
+  - push the tag to remote origin
+  - create a zip file of the `dist` directory
+
+13. Create a release in the [Github interface](https://github.com/alphagov/govuk-frontend/releases/new)
+  - select the latest tag version
+  - set "GOV.UK Frontend release v[version-number]" as the title
+  - add release notes from changelog
+  - attach the generated ZIP that is located at the root of the project 
+  - publish release
+
+14. (Required for private beta) Grant "test" user access to any newly published package(s).
+```bash
+npm access grant read-only govuk-frontend:test @govuk-frontend/[component-name]
+```
+15. Log out from npm
+```bash
+npm logout
 ```
-#### Test published packages
-
-1. To test the packages have published correctly, in console login as the test
-user. In the Design System, run `npm install` and then `npm start`. Check that
-the changes have taken effect.
-
-## Updating other repos that consume `govuk-frontend`
-
-If you need to update `govuk-frontend` to the latest release on another repo
-that consumes it, make sure you are logged in as the npm test user to ensure that
-the permissions of `govuk-frontend` packages have been correctly set.
 
-You can use `npm whoami` to check your current user.
+16. Add Trello cards to "This Sprint" column for
```

Comment:
Not sure about this step, I see it more as part of our release plan/process rather than the publishing process.

Reply:
leaving it there for now until there is a better home